packages/react-components/src/components/styles/RichTextEditor.styles.ts

```diff
@@ -54,6 +54,9 @@ export const richTextEditorWrapperStyle = (theme: Theme, addTopOffset: boolean):
           verticalAlign: 'top'
         }
       }
+    },
+    '& img': {
+      width: '119px'
```

Comment:
Do we stillneed this? (as it's already set in styles in CopyPastePlugin and it has higher specificity)

Reply:
> Do we stillneed this? (as it's already set in styles in CopyPastePlugin and it has higher specificity)

We still need this because this is the one that's actually send to the server and this is the one that set the size of the display in the RTE box. Currently the value are the same but in the upcoming stories, the one in the CopyPastePlugin should be the actual image size while this one will be 119px based on the design spec.